LICENSE.txt

```diff
@@ -2,6 +2,7 @@
 This project is available under the following license:
 ************************************
 Copyright 2012-2016 eBusiness Information
+Copyright 2016-2017 the AndroidAnnotations project
```

Comment:
This is not needed, this change would make the file the same as `header_both.txt`. This header is for files that are not yet touched since the project transfer.